src/Worker/Grpc/Logs.cs

```diff
@@ -57,5 +57,8 @@ static partial class Logs
 
         [LoggerMessage(EventId = 58, Level = LogLevel.Information, Message = "Abandoning orchestration. InstanceId = '{instanceId}'. Completion token = '{completionToken}'")]
         public static partial void AbandoningOrchestrationDueToVersioning(this ILogger logger, string instanceId, string completionToken);
+
+        [LoggerMessage(EventId = 59, Level = LogLevel.Information, Message = "Abandoning orchestration due to filtering. InstanceId = '{instanceId}'. Completion token = '{completionToken}'")]
+        public static partial void AbandoningOrchestrationDueToOrchestrationFilter(this ILogger logger, string instanceId, string completionToken);
```

Comment:
Should we include information about why it was filtered for debugging purposes? For example, because it matched name "Foo" or tag "Bar"?

Reply:
We won't actually know what it matched as that happens in the user implementation of the interface.